build.proj

```diff
@@ -26,8 +26,9 @@ of patent rights can be found in the PATENTS file in the same directory.
 		<PackageAssemblies Include="React.Core" />
 		<PackageAssemblies Include="React.MSBuild" />
 		<PackageAssemblies Include="React.Owin" />
-		<PackageAssemblies Include="React.Web" />
-		<PackageAssemblies Include="React.Web.Mvc4" />
+    <PackageAssemblies Include="React.Router" />
```

Comment:
Whoops, i went over the .cs files and tabified but i missed this file and some xml. fixed